extensions/integration-tests/src/objectExplorer.test.ts

```diff
@@ -8,17 +8,33 @@
 import 'mocha';
 import * as azdata from 'azdata';
 import { context } from './testContext';
-import { getDefaultTestingServer } from './testConfig';
+import { getDefaultTestingServer, getBdcServer, TestServerProfile } from './testConfig';
 import { connectToServer } from './utils';
 import assert = require('assert');
 
 if (context.RunTest) {
-	suite('Object Explorer integration test suite', () => {
+	suite('Object Explorer integration suite', () => {
+		test('BDC instance node label test', async function () {
+			const expectedNodeLable = ['Databases', 'Security', 'Server Objects', 'Data Services'];
+			let server = await getBdcServer();
+			await VerifyOeNode(server, 6000, expectedNodeLable);
```

Comment:
where do the numbers `6000` and `3000` come from?  Is there a more reliable way to determine when things are ready?